Lib/test/test_enum.py

```diff
@@ -2730,6 +2730,19 @@ def cycle_enum():
         self.assertEqual(256, len(seen), 'too many composite members created')
 
 
+class TestEmptyAndWeirdString(unittest.TestCase):
+
+    def test_empty_string(self):
+        empty_abc = Enum('empty_abc', ('', 'B', 'C'))
```

Comment:
This test should fail with a `ValueError` as empty strings for names are not allowed (include check in line 154 in enum.py).